satpy/readers/li_l2_nc.py

```diff
@@ -13,18 +13,36 @@
 # You should have received a copy of the GNU General Public License
 # along with satpy.  If not, see <http://www.gnu.org/licenses/>.
 
-"""MTG Lighting Imager (LI) L2 unified reader.
+"""MTG Lightning Imager (LI) L2 unified reader.
 
 This reader supports reading all the products from the LI L2
 processing level:
 
-  * L2-LE
-  * L2-LGR
-  * L2-AFA
-  * L2-LEF
-  * L2-LFL
-  * L2-AF
-  * L2-AFR
+  * L2-LE Lightning Events
+  * L2-LEF Lightning Events Filtered
+  * L2-LFL Lightning Flashes
+  * L2-LGR Lightning Groups
+  * L2-AF Accumulated Flashes
+  * L2-AFA Accumulated Flash Area
+  * L2-AFR Accumulated Flash Radiance
+

```

Comment:
i'd like to have a bit more "meat" and documentation in here for the reader... something like:
"The unified LI L2 reader returns the data either as an 1-D array or as a 2-D array depenging on the product type."

Reply:
Done